aten/src/ATen/native/cuda/CUDALoops.cuh

```diff
@@ -66,27 +65,6 @@ static constexpr int launch_bound2 = 4;
 
 namespace at { namespace native {
 
-template<int N>
```

Comment:
Moved to `aten/src/ATen/native/cuda/Loops.cuh` for visibility in `aten/src/ATen/native/cuda/DistributionTemplates.h`.